website/static/js/share/utils.js

```diff
@@ -28,8 +32,13 @@ utils.formatNumber = function(num) {
     return num;
 };
 
-var loadingIcon = m('img[src=/static/img/loading.gif]',{style: {margin: 'auto', display: 'block'}});
+utils.loadingIcon = {
```

Comment:
This probably doesn't belong in `utils`. It is a component, so should go with other components and follow naming conventions of other components (i.e. use UpperCamelCase).